glusterfsd/src/glusterfsd.c

```diff
@@ -1919,7 +1881,7 @@ print_exports_file(const char *exports_file)
  * Or if we cannot allocate anymore memory, we don't want to continue. Also,
  * we want to print out a different error messages based on the ret value.
  */
-int
+static int
```

Comment:
nit (and probably can go to a different patch) - this is only relevant if BUILD_GNFS is enabled, so perhaps can be ifdef'ed nicely.

Reply:
Thanks, will check it as well. And voting for the separate patch too.